deployments/local-setup-environment/provisioning/install-truffle.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+echo "Installing Truffle..."
+
+npm install -g truffle@5.0.30
+
+if ! [ -x "$(command -v truffle)" ]; then echo "Truffle installation failed"; exit 1; fi
+
+echo "Truffle has been installed successfully!"
```

Comment:
We have a lot of `install-*` scripts in `provisioning`. I wonder if we can kill two birds with one stone and integrated them into `linux-setup.sh`, put into https://github.com/keep-network/keep-core/tree/master/scripts and reference from here?

Reply:
We can do a rough copy paste and merge those scripts into one `linux-setup.sh`. Sharing those scripts is not so simple. First, from the Packer perspective, it's better to have multiple small scripts, because it's easier to customize the build process and errors are presented in a cleaner way. Also, if we move them to `keep-core`, we will be forced to clone this repo at the beginning of provisioning process and triggering them will be much more complicated.

Reply:
Alright, makes sense. If this setup is not Ubuntu-specific, let's prepare `linux-setup.sh` for `keep-core` since we are already here but we can keep those scripts separate here if that's what Packer prefers.

Reply:
Opened https://github.com/keep-network/keep-core/pull/1939. I named it `ubuntu-setup.sh` because I didn't tested it on other distros so I think that `linux-setup.sh` is rather inadequate.